.github/workflow_templates/e2e-daily.yml

```diff
@@ -0,0 +1,51 @@
+{!{- $enableWorkflowOnTestRepos := false -}!}
+{!{- $workflowName := "Daily e2e tests" -}!}
+name: '{!{ $workflowName }!}'
+on:
+  schedule:
+  - cron: '5 21 * * *'
+  workflow_dispatch:
+
+env:
+  WERF_DRY_RUN: "false"
+{!{ tmpl.Exec "werf_envs" | strings.Indent 2 }!}
+
+# Always run a single job at a time.
+# Note: Concurrency is currently in beta and subject to change.
+# https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#concurrency
+concurrency:
+  group: e2e-daily
```

Comment:
I think it is ok to run e2e tests concurrently in multiple cloud? Isn't it true? Otherwise we will not be able to add Docker or more kubernetes versions to these tests.

Reply:
'Always run a single job at a time.' — term 'job' is not correct here. Jobs in workflow are running in parallel. This settings are for workflow runs. `group: e2e-daily` is ok.